dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -704,6 +704,12 @@
 			interrupts = <106 0>;
 			status = "disabled";
 		};
+
+		fmc: fmc@44020000 {
+			compatible = "st,stm32-fmc";
+			reg = <0x44020000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00000001>;
+		};
```

Comment:
Please add `status = "disabled";`.
It may help fixing the CI issue you're facing (missing mandatory `pinctrl-0`)

Reply:
Thank you !  This fix it.
I couldn't understand the error on the CI so I had no idea where to start looking.